src/rules/semicolonRule.ts

```diff
@@ -58,6 +58,8 @@ export class Rule extends Lint.Rules.AbstractRule {
 
     public static FAILURE_STRING_MISSING = "Missing semicolon";
     public static FAILURE_STRING_UNNECESSARY = "Unnecessary semicolon";
+    public static FIX_STRING_MISSING = "Add semicolon";
```

Comment:
The idea was that rules could suggest more than one fix (though I can't actually think of a good example for needing more than one). Should each rule return at most one fix?